README.md

```diff
@@ -1,9 +1,11 @@
 <!--
-SPDX-FileCopyrightText: Open Energy Transition gGmbH and contributors to PyPSA-Eur <https://github.com/pypsa/pypsa-eur>
+SPDX-FileCopyrightText: Contributors to Open-TYNDP <https://github.com/open-energy-transition/open-tyndp>
+SPDX-FileCopyrightText: Contributors to PyPSA-Eur <https://github.com/pypsa/pypsa-eur>
```

Comment:
```suggestion
```

nit: If you have rewritten the whole README (which I think you have), then the PyPSA-Eur contributors have no place to be mentioned